test/unit/__snapshots__/expressions.spec.ts.snap

```diff
@@ -36,6 +36,116 @@ ____exports.__result = 10 - (4 + 5)
 return ____exports"
 `;
 
+exports[`Bitop [5.0] ("~a"): code 1`] = `
+"local ____exports = {}
+____exports.__result = bit.bnot(a)
+return ____exports"
+`;
+
+exports[`Bitop [5.0] ("~a"): diagnostics 1`] = `"main.ts(1,25): error TSTL: Bitwise operations is/are not supported for target Lua 5.1."`;
+
+exports[`Bitop [5.0] ("a&=b"): code 1`] = `
+"local ____exports = {}
+a = bit.band(a, b)
+____exports.__result = a
+return ____exports"
+`;
+
+exports[`Bitop [5.0] ("a&=b"): diagnostics 1`] = `"main.ts(1,25): error TSTL: Bitwise operations is/are not supported for target Lua 5.1."`;
```

Comment:
This snapshot or test seems to not be updated yet

Reply:
Fixed with 64e5f6d9f572ab04cd31895e5486bacaf354152e.